spec/system/ai_helper/ai_proofreading_spec.rb

```diff
@@ -0,0 +1,46 @@
+# frozen_string_literal: true
+
+RSpec.describe "AI Composer Proofreading Features", type: :system, js: true do
+  fab!(:user) { Fabricate(:admin) }
+
+  before do
+    Group.find_by(id: Group::AUTO_GROUPS[:admins]).add(user)
```

Comment:
No need for this --- change `Fabricate(:admin)` to `Fabricate(:admin, refresh_auto_groups: true)` above :)

Reply:
will clean this up.